Dockerfile.dapper

```diff
@@ -1,5 +1,6 @@
 FROM quay.io/costoolkit/releases-green:luet-toolchain-0.21.2 as luet
 FROM quay.io/costoolkit/releases-green:luet-makeiso-toolchain-0.4.0 as makeiso
```

Comment:
If we do not need luet-makeiso, we could remove all lines related to it. (include the `COPY` line below)